waspc/README.md

```diff
@@ -168,7 +168,7 @@ When done, new tab in your browser should open and you will see a Todo App!
 
 ### Note for team members
 
-Don't update e2e tests until someone approves the PR. They slow down the UI and create noise for the reviewers.
+Don't update e2e tests until your PR is approved. Running them and accepting changes prematurely can slow down the review process, create noise for reviewers, and makes it less likely that you'll thoroughly check the final diffs if changes are made during review iterations. **Carefully review the diffs before accepting them** (see [End-to-end (e2e) tests](#end-to-end-e2e-tests) for more details).
```

Comment:
Let's make it more authoritative and add spacing for emphasis.
I also corrected a minor mistake (e.g., review process -> UI)

```suggestion
Don not update e2e tests until your PR is approved.

Running the e2e tests and accepting them prematurely:
- Slows down the UI.
- Creates noise for the reviewers.
- Makes it less likely that you'll thoroughly check the final diffs after all review iterations.

**Carefully review e2e test diffs before accepting them** (see [End-to-end (e2e) tests](#end-to-end-e2e-tests) for more details).
```